### ISSUE TITLE
Adding Open Search (OpenSearch Specification) on the angular.io website

### ISSUE THREAD

**Issue by Davilink** [angular/angular#20609]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
</code></pre>

## Current behavior
Doesn't exist

## Expected behavior
Able to add the search engine in my browser to quickly search on the angular.io docs.

## Minimal reproduction of the problem with instructions
N/A

## What is the motivation / use case for changing the behavior?
Be able to quickly search in the documention, without needing to go on angular.io to click in the search bar to finally search what i'm looking for.

## Environment

<pre><code>
Angular version: X.Y.Z

Browser:
- N/A

Others:
the Open Search Specification website : http://www.opensearch.org/Home
</code></pre>

**Comment by gkalpak** [comment]
Duplicate of #19462.

**Comment by gkalpak** [comment]
**tl;dr**
PRs welcome :smiley:
You can still add it manually to your browser.

**Comment by Davilink** [comment]
Thx, i will see what i can do for a PR, but i can't add it manually with firefox, i will have to create a extension for the search engine.

**Comment by gkalpak** [comment]
You are right. I was referring to Chrome :grin:
